pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -377,7 +377,7 @@ func (e *TokensController) ensureReferencedToken(serviceAccount *v1.ServiceAccou
 	}
 	if liveServiceAccount.ResourceVersion != serviceAccount.ResourceVersion {
 		// Retry if our liveServiceAccount doesn't match our cache's resourceVersion (either the live lookup or our cache are stale)
-		klog.V(4).Infof("liveServiceAccount.ResourceVersion (%s) does not match cache (%s), retrying", liveServiceAccount.ResourceVersion, serviceAccount.ResourceVersion)
+		klog.V(4).InfoS("LiveServiceAccount.ResourceVersion (%s) does not match cache (%s), retrying", liveServiceAccount.ResourceVersion, serviceAccount.ResourceVersion)
```

Comment:
Wrong format ?

Reply:
Can you point out the non-compliance with the [document](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)?

Reply:
```suggestion
		klog.V(4).InfoS("LiveServiceAccount.ResourceVersion does not match cache, retrying", "liveResourceVersion", liveServiceAccount.ResourceVersion, "cacheResourceVersion", serviceAccount.ResourceVersion)
```